pkg/processor/test/suite/suite.go

```diff
@@ -87,6 +101,80 @@ func (suite *TestSuite) SetupTest() {
 	suite.TestID = xid.New().String()
 }
 
+// BlastHTTP is a stress test suite
+func (suite *TestSuite) BlastHTTP(request StressRequest) bool {
+
+	// Set deployOptions of example function "outputter"
+	deployOptions := suite.GetDeployOptions(request.FunctionName,
+		suite.GetFunctionPath(request.FunctionPath))
+
+	// Configure deployOptipns properties, number of MaxWorkers like in the default stress request - 32
+	deployOptions.FunctionConfig.Meta.Name = fmt.Sprintf("%s-%s", deployOptions.FunctionConfig.Meta.Name, suite.TestID)
+	deployOptions.FunctionConfig.Spec.Build.NuclioSourceDir = suite.GetNuclioSourceDir()
+	deployOptions.FunctionConfig.Spec.Build.NoBaseImagesPull = true
+	deployOptions.FunctionConfig.Spec.HTTPPort = 8080
+	defaultHTTPTriggerConfiguration := functionconfig.Trigger{
+		Kind:       "http",
+		MaxWorkers: 32,
+		URL:        ":8080",
+	}
+	deployOptions.FunctionConfig.Spec.Triggers = map[string]functionconfig.Trigger{"trigger": defaultHTTPTriggerConfiguration}
+
+	// Check for a specific handler, if found update deployOptions accordingly
+	if request.Handler != "" {
```

Comment:
I think this `if` is not needed. If string is empty, just copy the empty string no?